sound/soc/sof/ipc4-topology.c

```diff
@@ -1236,6 +1236,25 @@ static int snd_sof_get_nhlt_endpoint_data(struct snd_sof_dev *sdev, struct snd_s
 }
 #endif
 
+static int ipc4_set_fmt_mask(struct snd_mask *fmt, unsigned int bit_depth)
```

Comment:
This helper is duplicate here as ipc4-pcm.c needs special treatment for 'bit_depth of zero' case.